webpack.config.js

```diff
@@ -177,7 +182,24 @@ const cacheGroups = {
  */
 const appConfig = {
   mode: WEBPACK_MODE,
-  entry: {app: 'app'},
+  entry: {
+    /**
+     * Main Sentry SPA
+     */
+    app: 'app',
+
+    /**
+     * Legacy CSS Webpack appConfig for Django-powered views.
+     * This generates a single "sentry.css" file that imports ALL component styles
+     * for use on Django-powered pages.
+     */
+    sentry: 'less/sentry.less',
+
+    /**
+     * old plugins that use select2 when creating a new issue e.g. Trello, Teamwork*
```

Comment:
Is there a reason we can't just merge select2 into sentry's css, like does it make the bundle that much bigger?

Reply:
Is there a reason to merge it? We only include it in the plugins django template atm.

Reply:
really just to have less stuff in here. But yeah, that's fine.